src/blockchain/examples/simple_node.rs

```diff
@@ -156,9 +165,27 @@ async fn main() -> Result<(), Box<dyn Error>> {
     }
 }
 
+//Write a block to the file
+pub fn write_block(path: String, block: block::Block) {
+    use std::fs::OpenOptions;
+    use std::io::Write;
+
+    let mut file = OpenOptions::new()
+        .write(true)
+        .append(true)
+        .create(true)
+        .open(path)
+        .expect("cannot open file");
+
+    file.write_all(serde_json::to_string(&block).unwrap().as_bytes())
+        .expect("write failed");
+    file.write_all(b"\n").expect("write failed");
```

Comment:
And this - we want just json - and, really, we want this to be well formed JSON. But for now, we use as is

Reply:
This is JSON. Maybe I misunderstood.

Reply:
The blocks are json but the file format isn't. You're delimiting the blocks with new lines instead of putting them in an `[]`